gmrc/annotation/brain/util.py

```diff
@@ -0,0 +1,62 @@
+from pathlib import Path
+
+from rdflib import Graph, ConjunctiveGraph
+
+from typing import Dict, Iterable
+
+queries_path = Path(__file__).parent.resolve() / 'queries'
```

Comment:
Hm, there should be a more elegant way to access the files in the package. If I recall correctly import [importlib.resources](https://stackoverflow.com/questions/6028000/how-to-read-a-static-file-from-inside-a-python-package) could be an alternative. It probably doesn't matter here, but the motivation not to use the file path is that you the code could be packaged in some distribution format (egg or tar or whatever), and in those cases it is not so obvious anymore that this will work..

Reply:
yeah, managing python paths is not easy!

Note that as Thomas pointed out, using adding something like `__file__` or `parent` will work differently from how you installed the package (i.e. `pip install -e .` or `pip install .`)